extensions/amp-video/0.1/test/test-amp-video.js

```diff
@@ -165,13 +165,14 @@ describes.realWin('amp-video', {
     }, sources)).to.be.rejectedWith(/start with/);
   });
 
-  it('should set poster and controls in prerender mode', () => {
+  it('should set poster, controls, controlsList in prerender mode', () => {
     return getVideo({
       src: 'video.mp4',
       width: 160,
       height: 90,
       'poster': 'img.png',
       'controls': '',
+      'controlsList': 'nofullscreen nodownload noremote',
```

Comment:
isn't it `noremoteplayback` ?

Reply:
Yuu are right, source code (https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/html/media/HTMLMediaElementControlsList.cpp?q=noremote&sq=package:chromium&dr=CSs&l=15) says `noremoteplayback` this article (https://developers.google.com/web/updates/2017/03/chrome-58-media-updates) says `noremote` /cc @beaufortfrancois to update that article.

Reply:
Thank you so much @aghassemi! I'll update article now: https://github.com/google/WebFundamentals/pull/5042/